psi4/driver/qcdb/libmintsbasisset.py

```diff
@@ -352,7 +352,7 @@ def constructor_role_mol_shellmap(self, role, mol, shell_map, is_ecp = False):
                     self.uoriginal_coefficients[tst:tsp],
                     self.uexponents[tst:tsp],
                     'Pure' if self.puream else 'Cartesian',
-                    n, xyz_ptr, bf_count, pt='Normalised' if is_ecp else 'Unnormalized',
+                    n, xyz_ptr, bf_count, pt='Normalized' if is_ecp else 'Unnormalized',
```

Comment:
As a Englishman living in the States, I often interchange these spellings.  Anywhere else in the code it wouldn't matter but these keywords are used in comparison operations, so very strange bugs could result.  Thanks for fixing :)